### ISSUE TITLE
model.train() vs. eval()

### ISSUE THREAD

**Issue by jdunnmon** [HazyResearch/metal#70]
Confirm that model.train() and model.eval() are called in such a way that batchnorm/dropout behave correctly
